commerce-media-api/src/main/java/com/liferay/commerce/media/CommerceMediaDefaultImage.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.commerce.media;
+
+import aQute.bnd.annotation.ProviderType;
+
+import com.liferay.portal.kernel.exception.PortalException;
+
+/**
+ * @author Alec Sloan
+ */
+@ProviderType
+public interface CommerceMediaDefaultImage {
+
+	public long getConfiguration(long groupId) throws PortalException;
+
+	public void updateConfiguration(String fileEntryId, long groupId)
```

Comment:
we should try to think of a way to avoid passing `fileEntryId` here -- fileEntry is implementation-specific

Reply:
Multiple methods with different type options

groupId + {fileEntryId,inputStream,bytes}